osu.Game/Rulesets/UI/Playfield.cs

```diff
@@ -91,6 +99,7 @@ protected Playfield()
             {
                 h.NewResult += (d, r) => NewResult?.Invoke(d, r);
                 h.RevertResult += (d, r) => RevertResult?.Invoke(d, r);
+                h.DrawableHitObjectAdded += d => DrawableHitObjectAdded?.Invoke(d);
```

Comment:
I'm not sure about this, is there a case you can imagine for this to be used? I'd advise this to only be used for setup purposes and so it should correspond to the Playfield the DHO was added to.